.github/workflows/build-wheel.yml

```diff
@@ -135,16 +135,40 @@ jobs:
           pip install wheel
           pip install twine
 
-          # Download native artifacts
-          python scripts/download_artifacts.py $C2PA_VERSION
+          # Download native artifacts for the target architecture
+          if [ "${{ inputs.architecture }}" = "universal2" ]; then
```

Comment:
Why is it universal2? Is that a new standard or just some way to avoid conflicts?

Reply:
It is universal2 because "universal" was when mac did the switch from PowerPC to Intel. So universal flags PowerPC/Intel support. universal2 means Intel/ARM.